doc/developer-notes.md

```diff
@@ -1080,6 +1080,10 @@ A few guidelines for introducing and reviewing new RPC interfaces:
     until the wallet is caught up to the chainstate as of the RPC call's entry.
     This also makes the API much easier for RPC clients to reason about.
 
+- The `getbalances` RPC now returns `lastblock` which contains the wallet's last processed block
+  hash and height at the time the balances were calculated. As this call locks the chainstate we can also inform about the block
```

Comment:
"we can also inform about the balances" -> I don't quite understand what this means.

Reply:
I was pointing to this sentence from the original issue:

`
Those calls could benefit if the block hash they were valid on was included in the response. For example, immature balance can only change when the chainstate changes. 
`

Maybe I should reformulate the sentence.

Reply:
Have now removed the problematic sentence.